### PR TITLE
fix for issue #444: make CharString length check a warning

### PR DESCRIPTION
**Pull request by cjchapman**
(rather than a fatal error)

### CODE REVIEW

**Comment by miguelsousa** [comment]
@cjchapman I think I know why Travis didn't kick in. Looking into it now.

File: Tests/tx_test.py [hunk 1]
```diff
@@ -174,3 +174,11 @@ def test_remove_hints_bug180():
...
     expected_path = _get_expected_path('cid_nohints.ps')
     assert differ([expected_path, actual_path, '-m', 'bin'])
+
+
+def test_long_charstring_bug444():
+    # read a CFF2 VF with a charstring longer that 65535, check output
+    actual_path = runner(CMD + ['-o', '0', '-f', 'CJK-VarTest.otf'])
+    expected_path = _get_expected_path('CJK-VarTest.txt')
+    skip = ['-s', '## Filename']
+    assert differ([expected_path, actual_path] + skip)
```
Comment (miguelsousa) [inline Tests/tx_test.py]: you don't need to define `skip`

Comment (cjchapman) [reply Tests/tx_test.py]: I think I do. If I comment it out, I get:
```
    def test_long_charstring_bug444():
        # read a CFF2 VF with a charstring longer that 65535, check output
        actual_path = runner(CMD + ['-o', '0', '-f', 'CJK-VarTest.otf'])
        expected_path = _get_expected_path('CJK-VarTest.txt')
        # skip = ['-s', '## Filename']
>       assert differ([expected_path, actual_path] + skip)
E       NameError: global name 'skip' is not defined
```

Comment (miguelsousa) [reply Tests/tx_test.py]: sorry what I meant was, just do
```python
assert differ([expected_path, actual_path, '-s', '## Filename'])
```

**Pull request merged**